src/components/UserComboBox/UserComboBox.tsx

```diff
@@ -23,7 +23,7 @@ export const UserComboBox = ({ user, onChange }: UserComboBoxProps) => {
 
     return (
         <ComboBox
-            value={search}
+            value={search || user?.name}
```

Comment:
This will create a bug where you cannot erase the name. Do this instead

`    const [search, setSearch] = useState(user?.name || '');`

Probably should be fixed in other comboboxes also

Reply:
Thanks!